index.html

```diff
@@ -1,9 +1,9 @@
 <!DOCTYPE html>
-<html lang="en" dir="ltr" typeof="bibo:Document w3p:NOTE" about="" property="dcterms:language"
```

Comment:
A comment from who comes here from https://github.com/w3c/html/issues/588#issuecomment-254681667

A better example is

```
Specifying the language of content is useful for a wide number of applications,
from linguistically-sensitive searching to applying language-specific display properties.
In some cases the potential applications for language information are still waiting for implementations to catch up, whereas in others it is a necessity today.
Adding markup for language information to content is something that can and should be done as content is first developed.
If not, it will be much more difficult to take advantage of any future developments.
```

and not 

```
Specifying the language of content is useful for a wide number of applications, from
linguistically-sensitive searching to applying language-specific display properties.
In some cases the potential applications for language information are still
waiting for implementations to catch up, whereas in others it is a necessity
today. Adding markup for language information to content is something that
can and should be done as content is first developed. If not, it will be much
more difficult to take advantage of any future developments.
```

We have other texts that have a better line break by sentences, but this one, that was originally used as example, is not a good one.

A better diff is on this link https://github.com/webiwg/acessibilidade-web-pt/commit/a449b72a6adfc7590410490d942b33c87016dc59#diff-c7977ccbd401f475c5dff983d52c84d1R134